packages/next/package.json

```diff
@@ -72,9 +72,9 @@
     "assert": "2.0.0",
     "ast-types": "0.13.2",
     "browserify-zlib": "0.2.0",
-    "browserslist": "4.16.1",
+    "browserslist": "4.16.6",
     "buffer": "5.6.0",
-    "caniuse-lite": "^1.0.30001179",
+    "caniuse-lite": "1.0.30001219",
```

Comment:
`caniuse-lite` is a direct dependency of `browserslist`. I bumped the version to match `browserslist`’s expectations. Note that I also removed `^` to match other packages. Happy to revert if this was not a good guess! 🙂

Reply:
It's unpinned on purpose, see https://github.com/vercel/next.js/pull/16048

Reply:
Thanks for this extra context @merceyz – I unpinned the lib. Do you see any other potential issues? Spotting them earlier can reduce the overall review time.